### PR TITLE
Refactor/css

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Updated the styling for the modal popup, updated the styling on the navigation bar, created mixins and variables, created the color palette for dark mode, optimized website for tablet, optimized website for mobile, and passed Lighthouse and Wave accessibility audits 100%.

What (if anything) did you refactor?
- CSS mostly.

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/ArtDetails/ArtDetails.scss [hunk 1]
```diff
@@ -1,59 +1,70 @@
...
-  border-radius: 5px;
+  border-radius: $border-radius;
   text-transform: uppercase;
   font-weight: 900;
-  box-shadow: 5px 5px 0px rgba(0, 0, 0, .25);
-  cursor: pointer;
+  box-shadow: $shadow;
   text-decoration: none;
+  box-sizing: border-box;
+  cursor: pointer;
```
Comment (mr-ryan12) [inline src/ArtDetails/ArtDetails.scss:41]: Nice use of `@includes`

File: src/ArtDetails/ArtDetails.scss [hunk 2]
```diff
@@ -69,10 +80,59 @@ button {
...
+    font-size: 15px;
+  }
+
+  .main-details {
+    height: 70%;
+    width: 90%;
+    padding: 5%;
+  }
+}
+
```
Comment (mr-ryan12) [inline src/ArtDetails/ArtDetails.scss:109]: Mobile optimization looks fantastic! Thank you for doing this!

File: src/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+$secondary-color: #000;
+$invert-icon-amount: 0;
+$art-details-modal-color: #000000BB;
+
+//For Dark Mode, use the following settings:
+// $shadow: 5px 5px 0px rgba(255, 255, 255, 0.4);
+// $primary-color: #000;
+// $secondary-color: #fff;
+// $invert-icon-amount: 1;
+// $art-details-modal-color: #FFFFFF66;
```
Comment (mr-ryan12) [inline src/_variables.scss:19]: Thank you for getting this styling for dark mode. I'll look into some implementation!

File: src/ArtDetails/ArtDetails.scss [hunk 4]
```diff
@@ -1,59 +1,70 @@
...
-  border-radius: 5px;
+  color: $primary-color;
+  border-radius: $border-radius;
   width: 65%;
-  height: 74%;
+  height: 70%;
   margin-top: 5%;
   font-size: 30px;
   overflow-y: auto;
 }
```
Comment (mr-ryan12) [inline src/ArtDetails/ArtDetails.scss:25]: Thank you so much for DRYing up our SCSS with variables! Looks fantastic!

File: src/ArtDetails/ArtDetails.scss [hunk 5]
```diff
@@ -1,59 +1,70 @@
...
 
 .go-back-button,
 .generate-new-button {
-  background-color: black;
-  color: white;
-  // height: 18%;
-  width: 25%;
+  @include flexbox-center-center;
```
Comment (StevieAmb) [inline src/ArtDetails/ArtDetails.scss:29]: Your Sass skills are extraordinary.

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**